package.json

```diff
@@ -18,72 +18,85 @@
   "bugs": {
     "url": "https://github.com/getblocklab/block-lab/issues"
   },
-  "dependencies": {
-    "classnames": "^2.2.6"
-  },
   "devDependencies": {
-    "@testing-library/jest-dom": "5.1.1",
-    "@testing-library/react": "9.4.0",
-    "@wordpress/api-fetch": "3.10.0",
-    "@wordpress/babel-preset-default": "4.8.0",
-    "@wordpress/block-editor": "3.4.0",
-    "@wordpress/components": "8.5.0",
-    "@wordpress/compose": "3.9.0",
-    "@wordpress/data": "4.11.0",
-    "@wordpress/dependency-extraction-webpack-plugin": "2.1.0",
-    "@wordpress/dom": "2.8.0",
-    "@wordpress/editor": "9.9.0",
-    "@wordpress/element": "2.10.0",
+    "@testing-library/jest-dom": "5.5.0",
+    "@testing-library/react": "10.0.3",
+    "@wordpress/api-fetch": "3.14.0",
+    "@wordpress/babel-preset-default": "4.12.1",
+    "@wordpress/block-editor": "3.10.0",
+    "@wordpress/components": "9.5.0",
+    "@wordpress/compose": "3.14.0",
+    "@wordpress/core-data": "2.15.0",
+    "@wordpress/data": "4.17.0",
+    "@wordpress/dependency-extraction-webpack-plugin": "2.6.0",
+    "@wordpress/dom": "2.9.0",
+    "@wordpress/e2e-test-utils": "4.6.0",
+    "@wordpress/e2e-tests": "1.15.0",
+    "@wordpress/editor": "9.15.0",
+    "@wordpress/element": "2.13.1",
     "@wordpress/eslint-plugin": "3.3.0",
     "@wordpress/hooks": "2.8.0",
-    "@wordpress/html-entities": "2.5.0",
-    "@wordpress/i18n": "3.7.0",
-    "@wordpress/keycodes": "2.7.0",
-    "@wordpress/scripts": "6.1.1",
-    "@wordpress/server-side-render": "1.5.0",
-    "@wordpress/url": "2.13.0",
-    "autoprefixer": "9.7.3",
+    "@wordpress/html-entities": "2.7.0",
+    "@wordpress/i18n": "3.12.0",
+    "@wordpress/jest-console": "3.6.0",
+    "@wordpress/keycodes": "2.12.0",
+    "@wordpress/scripts": "9.0.0",
+    "@wordpress/server-side-render": "1.11.0",
+    "@wordpress/url": "2.14.0",
+    "autoprefixer": "9.7.6",
```

Comment:
I went a little overboard in updating all of these `@wordpress` packages, but figured we should test the latest versions.